docs/mmbn_style_names.md

```diff
@@ -0,0 +1,893 @@
+Move | SurgngStrkes | Surging Strikes
+Move | DragonEnergy | Dragon Energy
+Move | FreezngGlare | Freezing Glare
+Move | ThnderusKick | Thunderous Kick
+Move | GlacialLance | Glacial Lance
+Move | AstrlBarrage | Astral Barrage
+Move | PsyshieldBsh | Psyshield Bash
+Move | SprngtdeStrm | Springtide Storm
+Move | MystcalPower | Mystical Power
+Move | MountainGale | Mountain Gale
+Move | VictoryDance | Victory Dance
+Move | HeadlongRush | Headlong Rush
+Move | BitterMalice | Bitter Malice
+Move | TripleArrows | Triple Arrows
+Move | InfrnlParade | Infernal Parade
+Move | CeaslessEdge | Ceaseless Edge
+Move | BlekwndStorm | Bleakwind Storm
+Move | WildbltStorm | Wildbolt Storm
+Move | SndsearStorm | Sandsear Storm
+Move | LunarBlessng | Lunar Blessing
+Move | LastRespects | Last Respects
+Move | SpicyExtract | Spicy Extract
+Move | PoplatinBomb | Population Bomb
+Move | RevivlBlesng | Revival Blessing
+Move | KowtowCleave | Kowtow Cleave
+Move | ColisinCours | Collision Course
+Move | ElectroDrift | Electro Drift
+Move | ChilReceptin | Chilly Reception
+Move | ChillingWatr | Chilling Water
+Move | GigatonHammr | Gigaton Hammer
+Move | BlazngTorque | Blazing Torque
+Move | WickedTorque | Wicked Torque
+Move | NoxiusTorque | Noxious Torque
+Move | CombatTorque | Combat Torque
+Move | MagiclTorque | Magical Torque
+Move | MatchaGotcha | Matcha Gotcha
+Move | TeraStarstrm | Tera Starstorm
+Move | BurnngBulwrk | Burning Bulwark
+Move | MightyCleave | Mighty Cleave
+Move | TachyonCuttr | Tachyon Cutter
+Move | AllurngVoice | Alluring Voice
+Move | SuprcellSlam | Supercell Slam
+Move | PsychicNoise | Psychic Noise
+Move | MalignntChan | Malignant Chain
+Species | Dudunsprce | Dudunsparce
+Species | Corvisquir | Corvisquire
+Species | Corviknigh | Corviknight
+Species | Barraskewd | Barraskewda
+Species | Centiskorc | Centiskorch
+Species | Polteageis | Polteageist
+Species | Stonjourne | Stonjourner
+Species | Meowscarad | Meowscarada
+Species | Sqawkabily | Squawkabilly
+Species | Kilowatrel | Kilowattrel
+Species | Brmblghast | Brambleghast
+Species | ScreamTail | Scream Tail
+Species | BruteBonet | Brute Bonnet
+Species | FluttrMane | Flutter Mane
+Species | SlithrWing | Slither Wing
+Species | SndyShocks | Sandy Shocks
+Species | IronTreads | Iron Treads
+Species | IronBundle | Iron Bundle
+Species | IronJuguls | Iron Jugulis
+Species | IronThorns | Iron Thorns
+Species | RoarngMoon | Roaring Moon
+Species | IronVliant | Iron Valiant
+Species | WalkngWake | Walking Wake
+Species | IronLeaves | Iron Leaves
+Species | Ptchageist | Poltchageist
+Species | Fezndipiti | Fezandipiti
+Species | GouginFire | Gouging Fire
+Species | RagingBolt | Raging Bolt
+Species | IronBouldr | Iron Boulder
+Species | Crabminabl | Crabominable
+Species | Blacephaln | Blacephalon
+Species | Bsculegion | Basculegion
+Species | Flechinder | Fletchinder
+Type | Fight | Fighting
+Type | Electr | Electric
+Type | Psychc | Psychic
+Type | Stellr | Stellar
+
+# List
```

Comment:
Is there a reason why this second list is here when the first one is available above?

Reply:
There have been a few times where I had a table and I wished it was just a list. I don't feel strongly about the form that file takes, happy to change it.

Reply:
Please do :)
A list can be extracted by pasting the table in Excel/Google Spreadsheets and copying the respective column :)

Reply:
[`ad35903` (#5240)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5240/commits/ad35903d3b2b4214f93a85983e8536d80b2b2aca)